raiden/raiden_service.py

```diff
@@ -444,6 +444,22 @@ def expect_exchange(
         asset_manager = self.raiden.get_manager_by_asset_address(from_asset)
         asset_manager.transfermanager.exchanges[ExchangeKey(from_asset, from_amount)] = exchange
 
+    def get_channel_list(self, asset_address=None, partner_address=None):
+        if asset_address:
+            asset_manager = self.raiden.get_manager_by_asset_address(asset_address)
+            if partner_address:
+                return [asset_manager.partneraddress_channel[partner_address]]
+            return asset_manager.address_channel.values()
+        else:
+            channel_list = []
+            if partner_address:
+                for manager in self.raiden.managers_by_asset_address.values():
+                    channel_list.extend([manager.partneraddress_channel[partner_address]])
```

Comment:
What do we want to return if there is no channel for a given partner_address? Throw keyError or return empty list?